### ISSUE TITLE
Problem getting React Native project to recognize RNStripeTerminal

### ISSUE THREAD

**Issue by khuang-citylift** [theopolisme/react-native-stripe-terminal#33]
Hi, the issue that I am having is my project does not recognize the RNStripeTerminal object. The failure is on the line of import:

`import StripeTerminal from 'react-native-stripe-terminal';`

I have successfully linked 'react-native-stripe-terminal' to my react native project because I see the following in my Podfile:

`pod 'StripeTerminal', '1.0.3'
  pod 'RNStripeTerminal', :path => '../node_modules/react-native-stripe-terminal'`

The path is correct and I see the react-native-stripe-terminal folder in my node_modules.
I have also followed the iOS SDK set up instructions here: https://stripe.com/docs/terminal/sdk/ios#install

I am not sure what I could be missing and I am hoping to get some advice on how I can troubleshoot this further. Thank you!

**Comment by rfischer20** [comment]
@khuang-citylift Did you have any luck with this? We have been at a standstill with the same issue trying to get this moving forward.

**Comment by khuang-citylift** [comment]
@rfischer20 No, I did not get react-native-stripe-terminal working but I found a workaround and that was to use to react-native-webview. We ended up just rendering the webpage inside of the react native application.
https://github.com/react-native-community/react-native-webview

**Comment by rfischer20** [comment]
@khuang-citylift appreciate the update! Did not know you could access the devices through the webview - we'll check it out

**Comment by vivekvardhanadepu** [comment]
@khuang-citylift It has something to do with the ios target iirc. Change it to some higher version and see if it works

**Comment by RichSDavies** [comment]
> @rfischer20 No, I did not get react-native-stripe-terminal working but I found a workaround and that was to use to react-native-webview. We ended up just rendering the webpage inside of the react native application. https://github.com/react-native-community/react-native-webview

We have also decided to go down this route, having issues with an android device where it cannot communicate with reader, terminal is onm same network just wondering if you had seen this issue.  Works fine on emulator and when i run the js in a browser
